templates/components/wizard-confirmation-step.html

```diff
@@ -0,0 +1,19 @@
+<script type="text/x-template" id="eddbk-confirmation-step-template">
+    <div v-if="service && appointment">
+        <div class="edd-booking-wizard__info">
+            <p v-html="_('You are booking: <strong>%(service)s</strong> - %(duration)s appointment', appointment)"></p>
```

Comment:
What syntax is that for placeholders? AFAIK the translation function should only support positional placeholders, and not the named ones. Please confirm.